analysis/release/state_of_external.log

```diff
@@ -0,0 +1,384 @@
+config/config_r.r|1490
+config/config_stata.do|734
+config/configuration.py|1798
+config/scons.zip|647844
+paper_slides/.DS_Store|6148
+paper_slides/.sconsign.dblite|6329
+paper_slides/config_global.yaml|357
+paper_slides/configuration_test.py|2016
+paper_slides/run.py|458
+paper_slides/SConstruct|2691
+analysis/release/sconstruct.log|250
+analysis/release/state_of_inputs.log|40684
+analysis/release/state_of_repo.log|3365
+analysis/source/.DS_Store|6148
+config/scons/scons-configure-cache.py|5436
+config/scons/scons-LICENSE|1271
+config/scons/scons-README|6428
+config/scons/scons-time.py|48769
+config/scons/scons.py|6946
+config/scons/sconsign.py|18234
+paper_slides/input/plot.eps|15385
+paper_slides/input/provenance.txt|78
+paper_slides/input/table.txt|39
+paper_slides/release/sconstruct.log|250
+paper_slides/release/state_of_repo.log|4679
+analysis/build/descriptive/plot.eps|15385
+analysis/build/descriptive/sconscript_descriptive.log|99
+analysis/build/descriptive/table.txt|39
+analysis/build/prepare_data/data.txt|1988897
+analysis/build/prepare_data/sconscript_create_data.log|99
+analysis/release/descriptive/plot.eps|15385
+analysis/release/descriptive/sconscript_descriptive.log|99
+analysis/release/descriptive/table.txt|39
+analysis/release/lg/data.txt|1988897
+analysis/source/descriptive/descriptive.py|572
+analysis/source/descriptive/SConscript|489
+analysis/source/prepare_data/create_data.do|262
+analysis/source/prepare_data/create_data.m|214
+analysis/source/prepare_data/create_data.py|271
+analysis/source/prepare_data/create_data.r|261
+analysis/source/prepare_data/create_data.sh|81
+analysis/source/prepare_data/SConscript|1123
+config/scons/scons-local-2.5.1/scons-2.5.1-py2.7.egg-info|442
+paper_slides/build/paper/ondeck.pdf|18274
+paper_slides/build/paper/online_appendix.pdf|45354
+paper_slides/build/paper/paper.pdf|58962
+paper_slides/build/paper/sconscript.log|342
+paper_slides/build/paper/text.pdf|18274
+paper_slides/build/tables/sconscript.log|157
+paper_slides/build/tables/table.lyx|4094
+paper_slides/build/talk/sconscript.log|589
+paper_slides/build/talk/slides.pdf|29958
+paper_slides/release/paper/ondeck.pdf|18193
+paper_slides/release/paper/online_appendix.pdf|45354
+paper_slides/release/paper/paper.pdf|58961
+paper_slides/release/paper/sconscript.log|6473
+paper_slides/release/paper/text.pdf|18193
+paper_slides/release/talk/sconscript.log|589
+paper_slides/release/talk/slides.pdf|29971
+paper_slides/source/figures/figure.lyx|2214
+paper_slides/source/paper/ondeck.lyx|2522
+paper_slides/source/paper/online_appendix.lyx|3610
+paper_slides/source/paper/paper.lyx|5236
+paper_slides/source/paper/SConscript|677
+paper_slides/source/paper/text.lyx|2522
+paper_slides/source/paper/text.tex|1300
+paper_slides/source/tables/SConscript|233
+paper_slides/source/tables/table.lyx|4082
+paper_slides/source/talk/SConscript|237
+paper_slides/source/talk/slides.lyx|4697
+config/scons/scons-local-2.5.1/SCons/__init__.py|1634
```

Comment:
we definitely want to exclude this directory

Reply:
or maybe not I'm not that definite about it on second thoughts

Reply:
This record is just a result of me not knowing which external directory I should point to. I froze and went with `..`, the entire template. I think there's no actual value in recording the state of template before a run, other than as demonstration. But you're right at https://github.com/gslab-econ/template/pull/87#discussion_r137432001 that we can come up with a better demo.